.github/workflows/ci-advanced.yml

```
@@ -0,0 +1,352 @@
+  notify:
+    name: Notification
+    runs-on: ubuntu-latest
+    needs: [deploy-staging, deploy-production]
+    if: always()
+    steps:
+      - name: Notify deployment status
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          channel: '#deployments'
+          webhook_url: ${{ secrets.SLACK_WEBHOOK }}
```

actionlint (1.7.7)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "channel", "custom_payload", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "text", "username"

(action)